ui/src/api/index.ts

```diff
@@ -2,8 +2,10 @@ import { Hydra } from 'alcaeus/web'
 import { HydraResource } from 'alcaeus'
 import store from '@/store'
 import { APIError } from './errors'
+import { apiResourceMixin } from './mixins/ApiResource'
 
 const rootURL = process.env.VUE_APP_API_URL
+const segmentSeparator = '!!' // used to replace slash in URI to prevent escaping
```

Comment:
I chose the double exclamation mark arbitrarily as it has a close to nil risk of colliding with actual API resource identifiers. And the stand out well enough in the address bar...